splat.js

```diff
@@ -23,17 +23,17 @@ class Splatter {
   }
 
   /**
-   * Check to see if tokens <= splat.length, assign { splat, meta } into the
-   * `info` accordingly, and write to this instance.
-   *
-   * @param  {Info} info Logform info message.
-   * @param  {String[]} tokens Set of string interpolation tokens.
-   * @returns {Info} Modified info message
-   * @private
-   */
+     * Check to see if tokens <= splat.length, assign { splat, meta } into the
+     * `info` accordingly, and write to this instance.
+     *
+     * @param  {Info} info Logform info message.
+     * @param  {String[]} tokens Set of string interpolation tokens.
+     * @returns {Info} Modified info message
+     * @private
+     */
   _splat(info, tokens) {
     const msg = info.message;
-    const splat = info[SPLAT] || [];
+    const splat = info[SPLAT] || info.splat || [];
```

Comment:
Why look for the non-symbol key?

Reply:
Basically because it wasn't clear that splat shouldn't be evaluated as a property.  If ...
```
logger.log({
   level: 'error',
  message: 'This is super secret - hide it.'
});
``` 
is okay, then why couldn't we pass a splat property as well?  Maybe like:
```
logger.log({
   level: 'error',
  message: 'This is super secret - hide it.',
  splat: new Error()
});
``` 

I also included this because #46 was expecting this to work.